### ISSUE TITLE
Weird evaluation values.

### ISSUE THREAD

**Issue by cxzhou35** [mohammadasim98/met3r#9]
Thanks for release this great work, I am tring to evaluate the methods on my own data. But the average score is not in the ranges you reported in the README (Should be between 0.22 - 0.29 for MASt3R), any advice to fix this problem? 

Below are some related codes:

```python
class Met3R(torch.nn.Module):
    def __init__(self, img_size: int=None, backbone: str="mast3r"):
        super(Met3R, self).__init__()
        self.img_size = img_size
        self.backbone = backbone

        try:
            from met3r import MEt3R
        except ImportError:
            log(f"Please install met3r: {red('pip install met3r')}")
            raise ImportError("met3r is not installed")

        self.metric = MEt3R(
            img_size=self.img_size,
            use_norm=True,
            backbone=self.backbone,
            feature_backbone="dino16",
            feature_backbone_weights="mhamilton723/FeatUp",
            upsampler="featup",
            distance="cosine",
            freeze=True,
        ).cuda()

    def forward(self, img1: torch.Tensor, img2: torch.Tensor):
        assert img1.shape == img2.shape, "The shape of input images must be the same."
        # normalized to [-1, 1]
        img1 = img1 * 2 - 1
        img2 = img2 * 2 - 1

        out_h = img1.shape[1] // 16 * 16
        out_w = img1.shape[2] // 16 * 16

        img1 = F_img.resize(img1, (out_h, out_w), interpolation=F_img.InterpolationMode.BICUBIC, antialias=True)
        img2 = F_img.resize(img2, (out_h, out_w), interpolation=F_img.InterpolationMode.BICUBIC, antialias=True)
        inputs = torch.stack([img1, img2], dim=0)
        inputs = rearrange(inputs, 'v c h w -> 1 v c h w')
        inputs = inputs.cuda()

        score, *_ = self.metric(
            images=inputs,
            return_overlap_mask=False, # Default
            return_score_map=False, # Default
            return_projections=False # Default
        )

        # Should be between 0.22 - 0.29 for MASt3R
        # Should be between 0.30 - 0.35 for DUST3R
        score_avg = score.mean().item()

        # Clear up GPU memory
        torch.cuda.empty_cache()

        return score_avg

met3r_metric = Met3R(img_size=256, backbone='mast3r)
fix_img = read_image("./images/32/000000.png")
next_fix_img = read_image("./images/33/000000.png")
met3r_score = met3r_metric(fix_img, next_fix_img)
print(met3r_score)

>>> 0.48319220542907715
```

**Comment by cxzhou35** [comment]
"./images/32/000000.png":

<img width="1600" height="900" alt="Image" src="https://github.com/user-attachments/assets/0d50fe04-f1c0-4bfc-aaf3-af7dc068318d" />

"./images/33/000000.png":

<img width="1600" height="900" alt="Image" src="https://github.com/user-attachments/assets/0eec3461-3c43-4632-976f-a1a060517abd" />

**Comment by aviralchharia** [comment]
I think the obtained MEt3R score is justified. If you notice closely, the scene behind the singer changed a lot, which causes the inconsistency in the metric

**Comment by asdjklqwejkl** [comment]
# RGB range must be in [-1, 1]
